### ISSUE TITLE
V2.2.0 error

### ISSUE THREAD

**Issue by ierpe** [ThemeFuse/Unyson#368]
Hey, after the update I got this error : 

``` 
Call to a member function render_slider() on a non-object in /var/www/myproject/web/app/plugins/unyson/framework/extensions/media/extensions/slider/class-fw-extension-slider.php on line 458
```

**Comment by benburne** [comment]
I've updated to V2.2.0 this morning and received the same error. Please advise the best fix as soon as possible!

**Comment by sergiubagrin** [comment]
Hi @ierpe , @benburne 

We work on this issue right now.

**Comment by moldcraft** [comment]
I very sorry. The fix v2.2.1 is live now, please update.

To prevent such situations, please follow [this repository](https://github.com/ThemeFuse/Unyson-Extensions-Approval). For e.g. [Unyson 2.2.0 was there](https://github.com/ThemeFuse/Unyson-Extensions-Approval/issues/39).

**Comment by ierpe** [comment]
Thanks @moldcraft !
